hack/verify-dependencies.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
```

Comment:
> If this is to be long-lived, I need a strong argument to convince me that it should be written in Bash.

Commands are right there, it's lifted from upstream so it already exists, and its scriptable in less than 50 lines.